locales/en/app.ftl

```diff
@@ -443,6 +443,37 @@ no-results-blurb = Sorry, that breach is not in our database.
 all-breaches-headline = All breaches in {-product-name}
 search-breaches = Search Breaches
 
+# This string contains nested markup that is later used to style and link the text inside of it.
+# Please do not modify or remove "<a>", "</a>", "<span>" and "</span>".
+facebook-breach-note = <span>Your email doesn’t appear in this breach,
+  but your phone number may still be vulnerable.</span> Some of the accounts
+  compromised in the Facebook breach included phone numbers and other
+  personal information but not email addresses. If you have ever signed up
+  for a Facebook account -- even if you don’t use it now -- we recommend you
+  take these steps to protect yourself
+
+# This string contains nested markup that is later used to style and link the text inside of it.
+# Please do not modify or remove "<a>", "</a>", "<span>" and "</span>".
+facebook-breach-what-to-do-1 = <span>Change your profile information to private
+  in your Facebook privacy settings at
+  <a>https://www.facebook.com/me/about</a>.</span> During this breach, hackers
+  took profile information that was set as “open to the public” or “shared with
+  friends.” This information can be combined with other data to access even
+  more of your personal information and accounts.
+
+# This string contains nested markup that is later used to style and link the text inside of it.
+# Please do not modify or remove "<a>", "</a>", "<span>" and "</span>".
+facebook-breach-what-to-do-2 = <span>Change the passcode on your <a>mobile
```

Comment:
This seems very US specific? Never heard of passcode for mobile provider accounts, in Italy or Germany.

Reply:
Happy to change the term to be more international and applicable ... when you call your mobile carrier for customer support, how do they verify your identity?

Reply:
I've only used PIN codes for banking support, not for mobile carriers. For those, typically personal information is required, and sometimes you need to physically go into a shop.

As a non native speaker, English copy really shouldn't come from me, but it can be as generic as "Check with your phone carrier how to protect your account from fraudulent activities like SIM-swapping"